README.md

```diff
@@ -12,19 +10,36 @@ Dataset Insights maintains a pip package for easy installation. It can work in a
 
```

Comment:
I think >= 3.7 is too vague. We only test it supports 3.7 and 3.8 but not 3.9. Can we specify in this way: Python 3 (3.7 and 3.8)

Reply:
I am not sure whether pining to a particular version (3.7 and 3.8) would be good approach.